deploy/prepare_deployment.bat

```diff
@@ -4,6 +4,11 @@ for /f "tokens=2-4 delims=/ "  %%a in ("%date%") do (set MM=%%a& set DD=%%b& set
 set build_date=%YYYY%-%MM%-%DD%
 set git_rev=%APPVEYOR_REPO_COMMIT:~0,8%
 
+rem Default values because %bintray_deploy% is currently unused
+set pkg_name=master
+set git_rev=%APPVEYOR_REPO_COMMIT:~0,8%
```

Comment:
Isn't this a duplicate string thes same as 5?

Reply:
Yes, and I realized it at the same time as you. See my comment about that.